### PR TITLE
corrected regex to properly parse microseconds

### PR DESCRIPTION
**Pull request by goddenrich**
fixes issue #149

### CODE REVIEW

**Comment by yliaog** [comment]
please sign cla

**Comment by goddenrich** [comment]
> please sign cla

I did before submitting pr. A little confused why its not showing up. I have signed up with the same email address as used for git commits and my full name. should the name be the username for github (ie. goddenrich)?

**Comment by goddenrich** [comment]
I signed it

**Comment by goddenrich** [comment]
@yliaog I have tried everything I can with the CLA. Just raised a ticket at jira.linuxfoundation.org to try to get this resolved

**Comment by micw523** [comment]
Okay now you're good for the CLA.

File: config/dateutil_test.py [hunk 1]
```diff
@@ -22,24 +22,26 @@
...
 
-    def _parse_rfc3339_test(self, st, y, m, d, h, mn, s):
+    def _parse_rfc3339_test(self, st, y, m, d, h, mn, s, ms):
         actual = parse_rfc3339(st)
-        expected = datetime(y, m, d, h, mn, s, 0, UTC)
+        expected = datetime(y, m, d, h, mn, s, ms, UTC)
         self.assertEqual(expected, actual)
 
     def test_parse_rfc3339(self):
         self._parse_rfc3339_test("2017-07-25T04:44:21Z",
```
Comment (yliaog) [inline config/dateutil_test.py]: could you please add more test cases, that add microsecond to the existing test cases, like "2017-07-25T04:44:21.5Z"

File: config/dateutil.py [hunk 2]
```diff
@@ -57,8 +59,10 @@ def parse_rfc3339(s):
     dt = [0] * 7
     for x in range(6):
         dt[x] = int(groups[x])
+    us = 0
     if groups[6] is not None:
-        dt[6] = int(groups[6])
+        partial_sec = float(groups[6].replace(",", "."))
```
Comment (yliaog) [inline config/dateutil.py]: could you please add a test case with ','? thanks.

Comment (goddenrich) [reply config/dateutil.py]: I thought I already had one testing the comma! done now thanks.

**Comment by yliaog** [comment]
thanks. please squash the commits.

/lgtm

**Comment by goddenrich** [comment]
I squashed commits and removed some unnecessary spaces. Thanks

**Pull request merged**